src/commands/integration-test-install-go.yaml

```diff
@@ -0,0 +1,33 @@
+steps:
```

Comment:
I'd rename it to `machine-install-go` - I can see this command being used in different jobs as well.